spec/javascripts/saveSpec.js

```diff
@@ -255,4 +254,21 @@ describe('Saving a resource instance', function() {
     });
 
   });
+
+  describe('payload for save requests', function() {
+    var resource;
+    beforeEach(function() {
+      resource = Model.create({ name: 'foo', id: 12, subject: 'hello world!' });
+    });
+
+    it('saves requested fields when specified', function() {
+      resource.save({fields: ['name', 'id']});
```

Comment:
> I think that's always the case, since instances are shared between consumers, there's a chance the same model instance could've been modified by a different consumer.

But in that case it's the consumer's responsibility to call `save()` isn't it?